root/programs/VB/Frameworks/Infrastructure/CustomControl/RichClient/PropertyGrid/CheckType.vb

```diff
@@ -42,215 +42,215 @@ Imports System.Windows
+        ''' <summary>全角</summary>
+        Public Property IsZenkaku() As Boolean
+            Get
+                Return m_IsZenkaku
+            End Get
+            Set(ByVal value As Boolean)
+                m_IsZenkaku = Value
+            End Set
+        End Property
+        Private m_IsZenkaku As Boolean
+        ''' <summary>数値</summary>
+        Public Property IsNumeric() As Boolean
+            Get
+                Return m_IsNumeric
+            End Get
+            Set(ByVal value As Boolean)
+                m_IsNumeric = Value
+            End Set
+        End Property
+        Private m_IsNumeric As Boolean
+        ''' <summary>片仮名</summary>
+        Public Property IsKatakana() As Boolean
+            Get
+                Return m_IsKatakana
+            End Get
+            Set(ByVal value As Boolean)
+                m_IsKatakana = Value
+            End Set
+        End Property
+        Private m_IsKatakana As Boolean
+        ''' <summary>半角片仮名</summary>
+        Public Property IsHanKatakana() As Boolean
+            Get
+                Return m_IsHanKatakana
+            End Get
+            Set(ByVal value As Boolean)
+                m_IsHanKatakana = Value
+            End Set
+        End Property
+        Private m_IsHanKatakana As Boolean
+        ''' <summary>平仮名</summary>
+        Public Property IsHiragana() As Boolean
+            Get
+                Return m_IsHiragana
+            End Get
+            Set(ByVal value As Boolean)
+                m_IsHiragana = Value
+            End Set
+        End Property
+        Private m_IsHiragana As Boolean
+        ''' <summary>日付</summary>
+        Public Property IsDate() As Boolean
+            Get
+                Return m_IsDate
+            End Get
+            Set(ByVal value As Boolean)
+                m_IsDate = Value
+            End Set
+        End Property
+        Private m_IsDate As Boolean
 
-		#Region "比較処理"
+#Region "比較処理"
 
-		''' <summary>ハッシュを返す</summary>
-		''' <returns>ハッシュコード</returns>
-		''' <remarks>
-		''' 全メンバのハッシュコードのXORではなく、
-		''' ビットマスクの演算（boolなので）
-		''' </remarks>
-		Public Overrides Function GetHashCode() As Integer
-			Dim hc As Integer = 0
+        ''' <summary>ハッシュを返す</summary>
+        ''' <returns>ハッシュコード</returns>
+        ''' <remarks>
+        ''' 全メンバのハッシュコードのXORではなく、
+        ''' ビットマスクの演算（boolなので）
+        ''' </remarks>
+        Public Overrides Function GetHashCode() As Integer
+            Dim hc As Integer = 0
 
-			If Me.IsIndispensabile Then
-				hc += 1
-			End If
-			If Me.IsHankaku Then
-				hc += 4
-			End If
-			If Me.IsZenkaku Then
-				hc += 2
-			End If
-			If Me.IsNumeric Then
-				hc += 8
-			End If
-			If Me.IsKatakana Then
-				hc += 16
-			End If
-			If Me.IsHanKatakana Then
-				hc += 32
-			End If
-			If Me.IsHiragana Then
-				hc += 64
-			End If
-			If Me.IsDate Then
-				hc += 128
-			End If
+            If Me.IsIndispensable Then
+                hc += 1
```

Comment:
@yusukemaegawa

I've registered this to the [issue](https://github.com/OpenTouryoProject/OpenTouryo/issues/148#issue-130225430).